RecoEgamma/EgammaElectronAlgos/src/EgammaPixelMatchParamObjects.cc

```diff
@@ -0,0 +1,70 @@
+#include "RecoEgamma/EgammaElectronAlgos/interface/EgammaPixelMatchParamObjects.h"
+
+edm::ParameterSetDescription egPM::makeParamBinsDesc()
+{
+  edm::ParameterSetDescription binParamDesc;
+  auto binDescCases = 
+    "AbsEtaClus" >>
+    (edm::ParameterDescription<double>("xMin",0,true) and
+     edm::ParameterDescription<double>("xMax",3.0,true) and
+     edm::ParameterDescription<int>("yMin",0,true) and
+     edm::ParameterDescription<int>("yMax",9999,true) and
+     edm::ParameterDescription<std::string>("funcType","TF1:pol0",true) and
+     edm::ParameterDescription<std::vector<double> >("funcParams",{0.},true)) or
+    "AbsEtaCharge" >>
+    (edm::ParameterDescription<double>("xMin",0,true) and
+     edm::ParameterDescription<double>("xMax",3.0,true) and
+     edm::ParameterDescription<int>("yMin",-1,true) and
+     edm::ParameterDescription<int>("yMax",1,true) and
+     edm::ParameterDescription<std::string>("funcType","TF1:pol0",true) and
```

Comment:
is this a correct syntax?
I see only one implementation of readFuncStr, which is supposed to parse this and it accepts only ":=" as a separator